core/nginx/nginx.conf.template

```diff
@@ -74,6 +74,7 @@ http {
       proxy_set_header Authorization "Basic %(auth)s";
     }
     location = /%(urlhash)s/beaker/ {
+      autoindex off;
```

Comment:
Separate fix I guess, but Scott noticed directory listings were still accessible thanks to the click issue.